applications/openshift/general/alert_receiver_configured/rule.yml

```diff
@@ -10,6 +10,7 @@ description: |-
   to send alert notifications to external systems so that designate personnel can be alerted 
   in real time.
 
+instructions: |-
```

Comment:
This needs to be removed. The instructions aren't populated in the rule because the operator silently ignores errors parsing the instructions from the datastream:

```
{"level":"info","ts":"2023-04-21T16:26:14Z","logger":"profileparser","msg":"failed to render value for rule","err":"error while parsing variables into values: : template: :2:73: executing \"\" at <index .data \"alertmanager.yaml\">: error calling index: cannot index slice/array with type string","errVerbose":"template: :2:73: executing \"\" at <index .data \"alertmanager.yaml\">: error calling index: cannot index slice/array with type string\nerror while parsing variables into values: \ngithub.com/ComplianceAsCode/compliance-operator/pkg/utils.RenderValues\n\tgithub.com/ComplianceAsCode/compliance-operator/pkg/utils/xml2text.go:118\ngithub.com/ComplianceAsCode/compliance-operator/pkg/utils.GetInstructionsForRule\n\tgithub.com/ComplianceAsCode/compliance-operator/pkg/utils/parse_arf_result.go:430\ngithub.com/ComplianceAsCode/compliance-operator/pkg/profileparser.ParseRulesAndDo.func1\n\tgithub.com/ComplianceAsCode/compliance-operator/pkg/profileparser/profileparser.go:617\nruntime.goexit\n\truntime/asm_amd64.s:1598"}
```